pkg/lib/oauth/handler/handler_authz.go

```diff
@@ -25,15 +25,31 @@ import (
 
 //go:generate mockgen -source=handler_authz.go -destination=handler_authz_mock_test.go -package handler_test
 
+const (
+	CodeResponseType          = "code"
+	NoneResponseType          = "none"
+	SettingsActonResponseType = "settings_action"
```

Comment:
Name this `urn:authgear:params:oauth:response-type:settings-action`

Reply:
Note to self: Update SDKs after server pr stabilized